### DISCUSSION TITLE
Yahoo Hack Together Hackathon

### DISCUSSION THREAD

**Discussion by aklish** [yahoo/elide#1867] [General]
Yahoo is hosting an open source hackathon. Elide is one of the projects. Check out [https://yahoo.github.io/hacktogether/](https://yahoo.github.io/hacktogether/) for more details.

**Comment by aklish** [comment]
Elide overview/training video:

https://www.youtube.com/watch?v=WeFzseAKbzs

**Reply by Brutus5000** [reply]
Shame that the video ends up in 480p only :(

**Reply by aklish** [reply]
Good catch.  I'm seeing if they can bump it to 720p.
